app/src/main/java/com/github/se/project/ui/lesson/SelectedTutorDetails.kt

```diff
@@ -0,0 +1,379 @@
+  Card(
+      modifier =
+          Modifier.fillMaxWidth()
+              .padding(vertical = 8.dp)
+              .border(1.dp, MaterialTheme.colorScheme.primary, MaterialTheme.shapes.medium)
+              .testTag("commentCard"),
+      colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.surfaceVariant)) {
+        Column(
+            modifier = Modifier.padding(16.dp), verticalArrangement = Arrangement.spacedBy(8.dp)) {
+              Row(
+                  modifier = Modifier.fillMaxWidth(),
+                  horizontalArrangement = Arrangement.spacedBy(8.dp),
+                  verticalAlignment = Alignment.CenterVertically) {
+                    // Profile picture placeholder
+                    DisplayProfilePicture("studentProfilePicture")
+
+                    // Student details
+                    Column(modifier = Modifier.weight(1f)) {
+                      Text(
+                          text = "${student.firstName} ${student.lastName}",
+                          style = MaterialTheme.typography.titleMedium,
+                          modifier = Modifier.testTag("studentName"))
+
+                      Text(
+                          text =
+                              "Lesson booked on ${SimpleDateFormat("dd/MM/yyyy").format(comment.date.toDate())}",
+                          style = MaterialTheme.typography.bodyMedium,
+                          color = MaterialTheme.colorScheme.onSurfaceVariant,
+                          modifier = Modifier.testTag("studentCommentDate"))
+                    }
+
+                    // Grade
+                    Column(horizontalAlignment = Alignment.End) {
+                      DisplayRatingGrade(comment.grade.toDouble(), "studentRating")
+                    }
+                  }
+
+              HorizontalDivider(color = MaterialTheme.colorScheme.outlineVariant)
+
+              Text(
+                  text = comment.comment,
+                  style = MaterialTheme.typography.bodyMedium,
+                  modifier = Modifier.testTag("studentComment"))
+            }
+      }
+}
+
+@Composable
+private fun DisplayRatingGrade(grade: Double, testTag: String) {
+  Row {
+    Text(
+        text = "$grade",
+        style = MaterialTheme.typography.labelMedium,
+        color = MaterialTheme.colorScheme.primary,
+        modifier = Modifier.testTag(testTag + "Label"))
+    Icon(
+        imageVector = Icons.Default.Star,
+        contentDescription = "Rating",
+        tint = MaterialTheme.colorScheme.primary,
+        modifier = Modifier.size(16.dp).padding(start = 4.dp).testTag(testTag + "Icon"))
+  }
+}
+
+@Composable
+private fun DisplayProfilePicture(testTag: String) {
+  Surface(
+      modifier = Modifier.size(48.dp),
+      shape = MaterialTheme.shapes.medium,
+      color = MaterialTheme.colorScheme.primary.copy(alpha = 0.3f)) {
+        Icon(
+            imageVector = Icons.Default.Person,
```

Comment:
I suppose this is placeholder? Otherwise not much point in displaying a blank generic profile picture